Lib/test/test_import/__init__.py

```diff
@@ -1255,15 +1255,26 @@ class Spec2:
 
     def test_create_builtin(self):
         class Spec:
-            name = None
+            pass
         spec = Spec()
 
+        spec.name = "sys"
+        self.assertIs(_imp.create_builtin(spec), sys)
+
+        spec.name = "builtins"
+        self.assertIs(_imp.create_builtin(spec), builtins)
+
+        name = None
```

Comment:
Do you mean spec.name?

```suggestion
        spec.name = None
```

Reply:
Thanks!